### PR TITLE
Updated git docs

### PR DESCRIPTION
**Pull request by matthewroach**
Updated documentation for Git process

### CODE REVIEW

File: News Theme Development.md [hunk 1]
```diff
@@ -133,95 +133,86 @@ Merge into `canary` branch to publish to canary tag. Please reach out to arc blo
...
-# Rebase canary onto beta
-git rebase beta
-# Push the updated canary branch (--force is required or else it will fail)
-git push --force origin canary
-```
+1. Enusre beta is ready - All PR's/hotfixes made against beta are merged in
+2. `git pull origin beta`
+3. `git push origin beta:stable`
+4. To release blocks
+    * Clean your local folder and install dependencies - `npx lerna clean -y && rm -rf node_modules && npm i && npx lerna clean -y`
```
Comment (JackHowa) [inline News Theme Development.md:170]: nice

File: News Theme Development.md [hunk 2]
```diff
@@ -133,95 +133,86 @@ Merge into `canary` branch to publish to canary tag. Please reach out to arc blo
...
+    * Publish the blocks - `npx lerna publish --conventional-commits --conventional-graduate`
```
Comment (JackHowa) [inline News Theme Development.md:171]: if all of the blocks are not published, we've also had to go through force-publishing to ensure they are. conventional-commits afaik give us changelogs but we don't use this. the force publish gives us also more flexibility for minor, major and patch releases https://github.com/WPMedia/fusion-news-theme-blocks/wiki/Stable-Publish:-Package-Not-Found-Of-Latest-Version

Comment (JackHowa) [reply News Theme Development.md:171]: I might recommend the force publish @jgrosspietsch , thoughts?

File: News Theme Development.md [hunk 3]
```diff
@@ -133,95 +133,86 @@ Merge into `canary` branch to publish to canary tag. Please reach out to arc blo
...
+    * Make sure all blocks have been graduated or promoted.
+    * `git push origin stable`
+5. Merge `stable` back to `canary`
```
Comment (JackHowa) [inline News Theme Development.md:176]: 👍 , and I might add, to ensure that the version number is iterated forward from the lerna process

File: News Theme Development.md [hunk 4]
```diff
@@ -133,95 +133,86 @@ Merge into `canary` branch to publish to canary tag. Please reach out to arc blo
...
+
+Any environment with the `BLOCK_DIST_TAG=stable` will get the updated blocks on next deploy
```
Comment (JackHowa) [inline News Theme Development.md:178]: as well as any with the deprecated `latest` due to the github action. links to the github actions might be helpful on push https://github.com/WPMedia/fusion-news-theme-blocks/actions/workflows/stable-dist-tag.yml

File: News Theme Development.md [hunk 5]
```diff
@@ -133,95 +133,86 @@ Merge into `canary` branch to publish to canary tag. Please reach out to arc blo
...
+
+### Hotfix a stable issue
+
+If an issue is found in stable this is a production issue, and requires a hotfix follow these steps
+
+1. `git pull origin stable` - Pll remote branch to ensure you have most upto date code
+2. `git checkout -b RC-Hotfix-ticket-123` - Create a hotfix branch
+3. Open pull request of your branch and target it to the relevant branch - `RC-Hotfix-ticket-123` -> `stable`
+4. release feature branch changes as `@hotfix` from stable.
+    * `npx lerna publish --force-publish --preid hotfix --pre-dist-tag hotfix`
```
Comment (JackHowa) [inline News Theme Development.md:193]: nice

File: News Theme Development.md [hunk 6]
```diff
@@ -133,95 +133,86 @@ Merge into `canary` branch to publish to canary tag. Please reach out to arc blo
...
+* Open pull request of your branch and target it to the relevant branch - `RC-Hotfix-ticket-123` -> `rc`
 
-`npx lerna publish --force-publish --preid hotfix --pre-dist-tag hotfix`
+Once pull request is approved, merge into target branch **also** merge target branch back to `canary`.
 
-4. Create feature pack with @hotfix blocks to test
-5. Upon successful QA, graduate the “hotfix” release to latest in the same publish workflow
 
-`npx lerna publish --conventional-commits --conventional-graduate`
+![Themes Git Flow](themes-git-flow.png "diagram")
```
Comment (JackHowa) [inline News Theme Development.md:220]: thank you

**Review by JackHowa — DISMISSED** [review]
Great work. only added suggestions and clarifications

File: News Theme Development.md (hunk 3 above)
Comment (matthewroach) [reply News Theme Development.md:176]: Added a note for this

File: News Theme Development.md (hunk 4 above)
Comment (matthewroach) [reply News Theme Development.md:178]: Added details and links to allow access to relevant actions

**Comment by JackHowa** [comment]
looks great -- thanks for the additions

**Review by JackHowa — APPROVED** [review]
Looks good -- nice work

**Pull request merged**